src/rust/src/pkcs7.rs

```diff
@@ -290,11 +296,97 @@ fn smime_canonicalize(data: &[u8], text_mode: bool) -> (Cow<'_, [u8]>, Cow<'_, [
     }
 }
 
+#[cfg(not(CRYPTOGRAPHY_IS_BORINGSSL))]
+fn load_pkcs7_certificates(
+    py: pyo3::Python<'_>,
+    pkcs7: Pkcs7,
+) -> CryptographyResult<Vec<x509::certificate::Certificate>> {
```

Comment:
Small nit: Instead of returning a Vec, I'd suggest the `&pyo3::types::PyList`. This avoids a copy+malloc round trip.

See verify.rs l239 for an example of this

Reply:
fixed